app/importers/planning_applications_importer.rb

```diff
@@ -5,26 +5,35 @@ def initialize(local_authority_name:)
     @local_authority_name = local_authority_name.downcase
   end
 
- def call
-   import_planning_applications
- rescue StandardError => exception
-   log_exception(exception)
- end
+  def call
+    import_planning_applications
+  rescue StandardError => exception
+    log_exception(exception)
+  end
```

Comment:
It looks like github actions is configured to run rubocop, but shouldn't it have picked up on indentation issues?

Reply:
Agreed, 👍🏻. I think we should make Rubocop more strict in order to pick it

Reply:
There's virtually no rules. It's programming in madmax world. And the application I'm using is adding in extra space here and there.